test/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do we need a `tsconfig.json` in the test folder? Would it be possible to reuse the one in the root folder?

Reply:
It's possible to use the config from the root for tests as well, but then the tests will be compiled to the lib directory. I don't think that is a right thing to do when the tests can be run without being compiled.

Reply:
Perfect then, thanks for letting me know.